LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) <year> <copyright holders>
```

Comment:
can you do exactly same as in [here](https://github.com/JMI-CSI/jmi-csi.github.io/blob/master/LICENSE)